discojs/src/client/decentralized/secret_shares.ts

```diff
@@ -3,23 +3,23 @@ import { List, Range } from 'immutable'
 import * as tf from '@tensorflow/tfjs'
 import * as crypto from 'crypto'
 
-import { Weights, aggregation } from '../..'
+import { WeightsContainer, aggregation } from '../..'
 
 const maxSeed: number = 2 ** 47
 /*
 Return Weights in the remaining share once N-1 shares have been constructed (where N is number of ready clients)
  */
-function lastShare (currentShares: List<Weights>, secret: Weights): Weights {
+function lastShare (currentShares: List<WeightsContainer>, secret: WeightsContainer): WeightsContainer {
   if (currentShares.size === 0) {
     throw new Error('Need at least one current share to be able to subtract secret from')
   }
-  return aggregation.subtractWeights(List([secret, aggregation.sumWeights(currentShares)]))
+  return secret.sub(aggregation.sum(currentShares))
```

Comment:
maybe `subtract` to be extra clear?

Reply:
I like short names as long as they're readable-enough, especially for such primitive operations — just like TF.js proposes `add`, `sub`, `div`, `mul`, `mod` and `pow` operations on its tensors. This is highly subjective, so I can fall back to more explicit names if you prefer :)

Reply:
you're right, it's better to be consistent with these TF ones